packages/runtime/src/logicsig.ts

```diff
@@ -0,0 +1,173 @@
+import {
+  Account, decodeAddress, encodeAddress,
+  generateAccount, MultiSig, MultiSigAccount,
+  multisigAddress, signBytes, verifyBytes
+} from "algosdk";
+
+import { compareArray } from "../src/lib/compare";
+import { convertToString, stringToBytes } from "../src/lib/parsing";
+
+export class LogicSig {
```

Comment:
Is this a copy paste from SDK? Why do we need it here? Maybe we can use inheritance and derive from SDK class if needed?

Reply:
some parts are same as SDK, but others are according to interpreter.
- in SDK i was not sure if program is in compiled bytes or that hash can only be calculated using the same function 
- or we can get address (escrow address) without network configuration or there are other dependencies on network config.
- i will check whole sdk function independently and confirm if there is dependency

Reply:
![error](https://user-images.githubusercontent.com/26249903/105636377-25f34900-5e8e-11eb-8c3b-f7d10e24e66b.png)

confirmed, cannot use SDK function because of `compiled bytes`.

Reply:
OK, please add a comment to this class explaining why we can't use SDK type. 
PS: maybe we can still inherit and overwrite some functionality which is not working.

Reply:
Can we also add a doc string pointing out that the we are using the raw TEAL code as program, which is different from the SDK (as they use compiled bytes)?

Reply:
ok, 
we can inherit but the functionality which is not working, is when class decodes the compiled bytes and check it.
this is happening in constructor, so while inheriting i couldn't figure a way - without calling super constructor.